src/docs/arch-transaction.dox

```diff
@@ -92,7 +91,7 @@ it is not included in the latest checkpoint.
 
 @subsection Isolation
 
-Isolation is one of the important feature of database to decide whether one transaction can
+Isolation is one of the important features of database to decide whether one transaction can
```

Comment:
Sorry, I know i got you to adjust this but it should be "features of a database, which is used to determine whether on transaction..."